README.md

```diff
@@ -46,6 +46,7 @@ module.exports = {
 |**`cache`**|`{Boolean\|String}`|`false`|Enable file caching|
 |**`parallel`**|`{Boolean\|Number}`|`false`|Use multi-process parallel running to improve the build speed|
 |**`sourceMap`**|`{Boolean}`|`false`|Use source maps to map error message locations to modules (This slows down the compilation) ⚠️ **`cheap-source-map` options don't work with this plugin**|
+|**`minify`**|`{Function}`|`(file, uglifyOptions) => uglify.minify(file, uglifyOptions)`|Allows you to override default minify function|
```

Comment:
The second param should be named differently

Reply:
seconded. (this repo could really benefit from a webpack-defaults update)

Reply:
`uglifyOptions` -> `options`?

Reply:
`options` seems good

Reply:
@ooflorent maybe better minifyOptions, because it will be misleading between options for plugin and options for minification

Reply:
yeah I could see that being a point of possible confusion